QModManager/Injector.cs

```diff
@@ -93,8 +94,7 @@ internal void Remove()
 
                 // Remove backup file if it exists
                 string backupFilePath = Path.Combine(managedDirectory, "Assembly-CSharp.qoriginal.dll");
-                if (File.Exists(backupFilePath))
-                    File.Delete(backupFilePath);
+                if (File.Exists(backupFilePath)) File.Delete(backupFilePath);
```

Comment:
Unnecessary change.  
Even single line if-block statements are easier read when on their own line and indented.

Reply:
Addressed in commit 35f3801